channels/http.py

```diff
@@ -117,8 +116,8 @@ def __init__(self, scope, body):
             except (ValueError, TypeError):
                 pass
         # Body handling
-        self._stream = body
-
+        self._stream = stream
+        self._read_started = False
```

Comment:
I think there should be one `self._read_started = False` in `__init__` as in  [here](https://github.com/django/django/blob/755673e1bca7edb6bee7a958f40d9ae54d85d44c/django/core/handlers/wsgi.py#L96), but not as [here](https://github.com/django/channels/pull/124/files).